src/pages/task/taskTabs/logs/LogTypes.tsx

```diff
@@ -212,6 +212,6 @@ const StyledPre = styled.pre`
   border: 1px solid ${gray.light2};
   border-radius: ${size.xxs};
   font-size: ${fontSize.m};
-  overflow-x: scroll;
+  overflow: scroll hidden;
```

Comment:
there is a double vertical scrollbar on Firefox (you can look at a task on staging environment to verify this), so this is just a change to get rid of that